### ISSUE TITLE
Approaching train announce calling points

### ISSUE THREAD

**Issue by josh-the-tech** [davwheat/rail-announcements#259]
Hi,

Would it be possible to have the approaching train announcement also do the calling points? If I recall (I don't live in the UK anymore) when it used to be Phil and Celia at Woking, it would announce the calling points when the approaching train announcement was being done.

Thanks!

**Comment by GTRspotter** [comment]
Yes, it would be helpful to have an option to announce the calling points or the origin station. 
This video could be useful: https://www.youtube.com/watch?v=TMvuvcByCXU
